src/appleseed/renderer/modeling/camera/perspectivecamera.cpp

```diff
@@ -36,6 +36,7 @@
 #include "renderer/utility/paramarray.h"
 
 // appleseed.foundation headers.
+#include "foundation/image/image.h"
```

Comment:
Good catch. It's also missing `foundation/image/canvasproperties.h`.

Reply:
Credit to the compiler.